etc/docker/defaults/greeting.html

```diff
@@ -1,17 +1,31 @@
 <h1>Welcome to Inbucket</h1>
 
 <p>Inbucket is an email testing service; it will accept email for any email
-address and make it available to view without a password.</p>
+    address and make it available to view without a password.
+</p>
 
 <p>To view messages for a particular address, enter the username portion
-of the address into the box on the upper right and click <em>View</em>.</p>
+    of the address into the box on the upper right and click <em>View</em>.
+</p>
 
 <p>This instance of Inbucket is running inside of a <a
-  href="https://www.docker.com/" target="_blank">Docker</a> container.  It is
-configured to retain messages for a maximum of 3 days, and will enforce a limit
-of 300 messages per mailbox - the oldest messages will be deleted to stay under
-that limit.</p>
+      href="https://www.docker.com/" target="_blank">Docker</a> container.  It is
+    configured to retain messages for a maximum of 3 days, and will enforce a limit
+    of 300 messages per mailbox - the oldest messages will be deleted to stay under
+    that limit.
+</p>
 
-<p>Messages addressed to any recipient in the <code>@bitbucket.local</code>
-domain will be accepted but not written to disk.  Use this domain for load or
-soak testing your application.</p>
+<p>
+    Messages addressed to any recipient in the <code>@bitbucket.local</code>
+    domain will be accepted but not written to disk.  Use this domain for load or
+    soak testing your application.
+</p>
+
+<p> You can modify this greetings page by changing the Docker environment variable 'INBUCKET_WEB_GREETINGFILE'
```

Comment:
Can you limit the line-length of the new paragraph to match the rest of the file above?  It looks like your editor also stripped off the trailing newline at the end of the file.

Reply:
As requested I split the content over more lines (including some </br> for new lines in the resulting greetings.html file). And I added the newline back to the file.